book/cli/reth/node.md

```diff
@@ -712,6 +712,8 @@ Engine:
 
       --engine.caching-and-prewarming
           Enable cross-block caching and parallel prewarming
```

Comment:
possibly we want to change the name to `disable-caching-and-prewarming` and keep the default as false, see how it's done for discovery args